tests/browser/stn/stn.browser.js

```diff
@@ -69,71 +69,82 @@ function runTests () {
     let fiVal = 30
     let fidVal = 8
 
+    let xhr = new XMLHttpRequest()
+    xhr.open('GET', 'https://api.publicapis.org/entries')
+    xhr.send()
+    xhr.addEventListener('load', () => setTimeout(runTests, 1000))
+
```

Comment:
I just migrated this file to a jest test in a PR branch, and actual XHR requests don't work well there so let's rethink this